cnd/src/db/schema.rs

```diff
@@ -1,106 +1,66 @@
+// LocalSwapId and SharedSwapId are encoded as Text and named local_swap_id, and
+// shared_swap_id respectively.  swap_id (Integer) is always a foreign key link
+// to the `swaps` table.
 table! {
-   rfc003_bitcoin_ethereum_bitcoin_ether_request_messages {
```

Comment:
These should not be deleted (yet) I guess :)

Reply:
They are moved to `rfc003_schema.rs`.